pkg/sync/kubernetes/kubernetes_sync.go

```diff
@@ -2,16 +2,15 @@ package kubernetes
 
 import (
 	"context"
-	"errors"
 	"fmt"
 	"os"
 	"strings"
+	parallel "sync"
```

Comment:
nitpick but when the flagd sync package and `"sync"` are being imported in the same package it  has previously been imported as "msync", might cause confusion to change the name in this package, maybe?